tests/main/test_main_general.py

```
@@ -64,6 +67,53 @@
+            f"stderr:\n{result.stderr}"
+        )
+
+
+@pytest.mark.skipif(sys.version_info[:2] != (3, 12), reason="Pydantic 2.5.3 regression is verified on Python 3.12")
+def test_cli_runs_with_pydantic_253() -> None:
+    """Run the installed CLI with the minimum accepted Pydantic 2.x version."""
+    uv = shutil.which("uv")
+    if uv is None:  # pragma: no cover
+        pytest.skip("uv is required for the Pydantic 2.5.3 e2e test")
+
+    result = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)